tests/test_dask_image/test_imread/test_core.py

```diff
@@ -69,12 +69,19 @@ def test_tiff_imread(tmpdir, seed, nframes, shape, dtype):
 
     a = np.random.uniform(low=low, high=high, size=shape).astype(dtype)
 
-    fn = str(dirpth.join("test.tiff"))
-    with tifffile.TiffWriter(fn) as fh:
-        for i in range(len(a)):
+    fn_pattern = str(dirpth.join("test_%s.tiff"))
+    for i in range(shape[0]):
+        with tifffile.TiffWriter(fn_pattern % i) as fh:
```

Comment:
This looks as if the previous tests were writing a multi-page tiff to a single file, and the new test is writing many individual tiff files instead. Do I misunderstand this?

If so, that seems like a significant change to me, and one that reduces the amount of meaningful test coverage we have.

Reply:
Update: I've just tried this on my local machine and yes, I think there is a meaningful difference between these, they're not equivalent.